src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -17,7 +16,7 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(Person person) {
         return this.getKeywords().stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getName().fullName, keyword));
+                .anyMatch(keyword -> person.getName().fullName.toLowerCase().contains(keyword.toLowerCase()));
```

Comment:
Can keep in view to possibly abstract this method into the superclass FindCommand if we decide to extend the logic